lib/sample_uploader/utils/sesar_api.py

```diff
@@ -42,6 +43,7 @@ def retrieve_sample_from_igsn(igsn):
     url = '{}/{}'.format(_get_igsn_endpoint(), igsn)
 
     try:
+        time.sleep(1)
```

Comment:
I realized the NCBI server set a 3 query per second limit. I don't think SESAR has any limitations. But just in case for large queries. I added this before every request.